src/hooks/usePlaylist.ts

```diff
@@ -16,9 +17,20 @@ export default function usePlaylist(
   usePlaceholderData: boolean = true,
   limit?: number,
 ): UsePlaylistResult {
-  return useQuery(['playlist', playlistId, relatedMediaId], () => getPlaylistById(playlistId, relatedMediaId, limit), {
-    enabled: !!playlistId && enabled,
-    placeholderData: usePlaceholderData ? placeholderData : undefined,
-    retry: false,
-  });
+  return useQuery(
+    ['playlist', playlistId, relatedMediaId],
+    async () => {
```

Comment:
You can just add `transformMediaItem` in the `.then` of getPlaylistById

Reply:
Removed